flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/operators/windowing/EvictingWindowOperator.java

```diff
@@ -333,7 +333,7 @@ public void onProcessingTime(InternalTimer<K, W> timer) throws Exception {
 		}
 	}
 
-	private void emitWindowContents(W window, Iterable<StreamRecord<IN>> contents, ListState<StreamRecord<IN>> windowState) throws Exception {
+	private void emitWindowContents(W window, Iterable<StreamRecord<IN>> contents, ListState<StreamRecord<IN>> windowState, boolean purge) throws Exception {
```

Comment:
Maybe we could place the parameters each in a line, such as the constructor.

Reply:
Accept.